beacon-chain/rpc/beaconv1/config.go

```diff
@@ -17,11 +21,80 @@ func (bs *Server) GetForkSchedule(ctx context.Context, req *ptypes.Empty) (*ethp
 // Values are returned with following format:
 // - any value starting with 0x in the spec is returned as a hex string.
 // - all other values are returned as number.
-func (bs *Server) GetSpec(ctx context.Context, req *ptypes.Empty) (*ethpb.SpecResponse, error) {
-	return nil, errors.New("unimplemented")
+func (bs *Server) GetSpec(ctx context.Context, _ *ptypes.Empty) (*ethpb.SpecResponse, error) {
+	ctx, span := trace.StartSpan(ctx, "beaconV1.GetSpec")
+	defer span.End()
+
+	data := prepareConfigSpec()
+	return &ethpb.SpecResponse{Data: data}, nil
 }
 
 // GetDepositContract retrieves deposit contract address and genesis fork version.
 func (bs *Server) GetDepositContract(ctx context.Context, req *ptypes.Empty) (*ethpb.DepositContractResponse, error) {
 	return nil, errors.New("unimplemented")
 }
+
+func prepareConfigSpec() map[string]string {
+	data := make(map[string]string)
```

Comment:
How can we ensure that 
1.) when a person adds new config and this gets updated accordingly
2.) the spec name is correctly aligned

This is going to get fragmented. Can we utilize yaml tags as keys?

Reply:
Good point. We can use reflect to walk through the struct fields of the config for a future-proof solution